### PR TITLE
[MM-34411] Change appID to com.mattermost.zendesk

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
- Change appID to com.mattermost.zendesk
- Update `README.md` with new installation instructions
- Drop unused `manifest.ts`

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-34411

### CODE REVIEW

File: src/restapi/fHelp.ts [hunk 1]
```diff
@@ -57,7 +56,7 @@ function addBullet(text: string): string {
 }
 
 function addBulletSlashCommand(text: string): string {
-    return `* \`/${AppID} ${text}\`\n`;
+    return `* \`/zendesk ${text}\`\n`;
```
Comment (larkox) [inline src/restapi/fHelp.ts:59]: Maybe put this in a constant (commandTrigger) and use it also in the corresponding binding in the other PR?

**Review by larkox — APPROVED** [review]
LGTM, just a small nit.

File: README.md [hunk 2]
```diff
@@ -70,8 +71,14 @@ Zendesk Admins are able to view these subscriptions inside Zendesk via `Settings
...
-`/apps install --url http://<your-zendesk-app>/manifest.json --app-secret thisisthesecret`  
-`/apps install --url http://localhost:4000/manifest.json --app-secret thisisthesecret`  
+### In Production
+
+`/apps install --app-id com.mattermost.zendesk`
+
+### In Development
+
+`/apps debug-add-manifest --url http://localhost:4000/manifest.json`
+`/apps install --app-id com.mattermost.zendesk--app-secret thisisthesecret`
```
Comment (jfrerich) [inline README.md:81]:
```suggestion
`/apps install --app-id com.mattermost.zendesk --app-secret thisisthesecret`
```

File: src/restapi/fHelp.ts (hunk 1 above)
Comment (jfrerich) [reply src/restapi/fHelp.ts:59]: I agree with putting this in a `commandTrigger` constant.

**Review by jfrerich — APPROVED** [review]
Thanks @hanzei!  One small nit and also suggest creating the constant suggest by Daniel.

File: src/restapi/fHelp.ts (hunk 1 above)
Comment (hanzei) [reply src/restapi/fHelp.ts]: Good suggestion :+1:

**Pull request merged**